### PR TITLE
Responsive Desktop Website

### PR DESCRIPTION
**Pull request by Kwaku28**
This is a full desktop website version
Sections added:
1. Portfolio: Indicating an introduction and worked on projects
2. About: Comprising of skills and experience with programming languages
3. Contact: Reach out form submitted to the website owner

Transitions (eg. scale) added to give a professional touch and user friendly.
Responsive to fit smaller screens.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,959 @@
...
+}
+
+.frame {
+  display: grid;
+  grid-template-columns: 369px 369px 369px;
+  justify-content: space-between;
+  padding-top: 50px;
+  gap: 24px;
+}
+
```
Comment (Lizdev-05) [inline style.css]: Great job so far 👍,  kindly check the responsiveness of your frames, it is breaking. 👇
image

File: style.css [hunk 2]
```diff
@@ -0,0 +1,959 @@
...
+.about-me {
+  display: flex;
+  flex-direction: column;
+  padding-top: 130px;
+  padding-left: 100px;
+  background: #1c1a19;
+}
+
```
Comment (Lizdev-05) [inline style.css]: Kindly the styling of your about-section, it is currently overflowing🙂
image

File: style.css [hunk 3]
```diff
@@ -0,0 +1,959 @@
...
+  background-image: url("./images/Illustration-header.png");
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 486px;
+  height: 500px;
+  position: absolute;
+  left: 66.5%;
+  top: 62px;
+}
+
```
Comment (Lizdev-05) [inline style.css]: Great job adding the background image 👍, kindly update the styling as it is currently not responsive and also, the background image is overflowing 🙂
image

**Review by Lizdev-05 — CHANGES_REQUESTED** [review]
Hi @Kwaku28, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- No linter errors ✔️
- Correct Github flow ✔️
- Descriptive PR ✔️
- Professional README file ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Favourezeugwa — APPROVED** [review]
## STATUS: APPROVED 🥇 🏆 🍹 

Hi @Kwaku28 , 

I am Favour 👩🏽 , your reviewer for this Pull Request.

Great Job implementing all the required changes suggested by the previous code reviewer. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Yayyy! 💃🏽, Congratulations! 🎉 🎉🎉

happy

## Highlights 💯 
- No Linter errors ✔️
- Great design/UI ✔️
- Good commit messages ✔️
- Professional Readme file ✔️
- Descriptive PR ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**